src/modules/localization.json

```diff
@@ -177,6 +177,38 @@
                     }
                 }
             ]
+        },
+        {
+            "name": "timeZone",
+            "tags": [
+                {
+                    "name": "property"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:localization:time-zone"
+                    ]
+                }
+            ],
+            "summary": "Set the IANA timezone for the device",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "$ref": "https://meta.comcast.com/firebolt/localization#/definitions/TimeZone"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": "America/New_York"
+                    }
+                }
+            ]
```

Comment:
Continuing discussion on getter APIs: Do we want to add getters for configured timezone and available timezones to the management SDK? Should apps be required to get timezone from the browser? What about native apps? If the core SDK doesn't provide an API is there inconsistency with other geolocation firebolt APIs?